src/parser/rogue/shared/constants.ts

```diff
@@ -1,4 +1,39 @@
+import SPELLS from 'common/SPELLS';
+import { SpellList } from 'common/SPELLS/Spell';
+
+export const SUBTLETY_ABILITY_COOLDOWNS: SpellList = {
+  ...SPELLS.SHADOWSTEP,
+  ...SPELLS.SHADOW_DANCE,
+  ...SPELLS.SYMBOLS_OF_DEATH,
+  ...SPELLS.SHADOW_BLADES,
+};
+
+export const ASSASSINATION_ABILITY_COOLDOWNS: SpellList = {
+  ...SPELLS.SHADOWSTEP,
+  ...SPELLS.VENDETTA,
+};
+
+export const OUTLAW_ABILITY_COOLDOWNS: SpellList = {
+  ...SPELLS.BETWEEN_THE_EYES,
+  ...SPELLS.BLADE_FLURRY,
+  ...SPELLS.ROLL_THE_BONES,
+  ...SPELLS.ADRENALINE_RUSH,
+  ...SPELLS.GRAPPLING_HOOK,
+};
+
+export const SHARED_ABILITY_COOLDOWNS: SpellList = {
+  ...SPELLS.CRIMSON_VIAL,
+  ...SPELLS.SPRINT,
+  ...SPELLS.KIDNEY_SHOT,
+  ...SPELLS.EVASION,
+  ...SPELLS.SHIV,
+  ...SPELLS.FEINT,
+  ...SPELLS.BLIND,
+  ...SPELLS.CLOAK_OF_SHADOWS,
+  ...SPELLS.TRICKS_OF_THE_TRADE,
+  ...SPELLS.SHROUD_OF_CONCEALMENT,
+};
 //region Legendaries
```

Comment:
isn't it easier to simply spread the SHARED_ABILITY_COOLDOWNS etc, when you're using them in the array?

Reply:
If I put them in an array then I'll have an array of an array and I'm not wanting that. I can possibly refactor this so that each spec has the SHARED cooldowns and then grab the proper const based on the spec being used from the module instead of using a switch-case.

Reply:
Think I came up with a better solution that what I had using only the class constants that include the shared spells in an array in constants.